utils/subprocess/command_wrapper_windows.go

```diff
@@ -0,0 +1,16 @@
+//go:build windows
+// +build windows
+
+package subprocess
+
+import (
+	"os/exec"
+	"syscall"
+)
+
+func setGroupAttrToCmd(c *exec.Cmd) {
+	c.SysProcAttr = &syscall.SysProcAttr{
+		HideWindow:    true,
+		CreationFlags: syscall.CREATE_UNICODE_ENVIRONMENT | syscall.CREATE_NEW_PROCESS_GROUP,
```

Comment:
Could you comment a link to docs for these?

Reply:
I added a link to Windows reference doc on this